packages/layout/src/tree-layout/drawing-manager/index.js

```diff
@@ -0,0 +1,53 @@
+import { Utils } from '../utils';
+import { drawLayout, resolveAligment, drawComponent } from './helper';
+
+export class DrawingManager {
+    constructor (data, renderer, container) {
+        this._data = data.tree;
+        this._componentMap = data.componentMap;
+        this._layoutClassName = data.layoutClassName;
+        this._renderer = renderer;
+        this._mount = Utils.isDOMElement(container) ? container : Utils.getElement(container);
+    }
+
+    draw () {
+        drawLayout(this);
+        resolveAligment(this, this._data);
+        drawComponent(this._data);
+    }
+
+    data (data) {
+        if (data) {
+            this._data = data;
+        }
+        return this._data;
+    }
+
+    componentMap (param) {
+        if (param) {
```

Comment:
This condition will fail if the value of param = 0
it's better to take params like (...params), check params.length and then use params[0] to initialize

use the generateGetterSetter utility function

Reply:
Also, dont use "params" for the argument name. Give a logical name